src/displayapp/screens/WatchFaceTerminal.h

```diff
@@ -30,13 +31,15 @@ namespace Pinetime {
                           Controllers::NotificationManager& notificationManager,
                           Controllers::Settings& settingsController,
                           Controllers::HeartRateController& heartRateController,
-                          Controllers::MotionController& motionController);
+                          Controllers::MotionController& motionController,
+                          Controllers::SimpleWeatherService& weatherService);
         ~WatchFaceTerminal() override;
 
         void Refresh() override;
 
       private:
         Utility::DirtyValue<int> batteryPercentRemaining {};
+        Utility::DirtyValue<uint8_t> hue {};
```

Comment:
I think this can be a local variable inside the function, no need to track its state at the class level

Reply:
Fixed.